Scanner fails with "unknown shorthand flag: 'X' in -Xmx5120m"

Running the action leads to the following error:

`unknown shorthand flag: 'X' in -Xmx5120m`

This seems to be a JDK flag. Here is the relevant part. See the attachment for the full log:

```yml
- name: Run GitHub Action for ORT
  uses: oss-review-toolkit/ort-ci-github-action@0650a3c52c02ad7366b8e25aeedf752d5601c417
  with:
    allow-dynamic-versions: 'true'
    ort-cli-args: '-P ort.analyzer.enabledPackageManagers=Nuget'
    run: >
      analyzer,
      scanner,
      evaluator,
      reporter,
      upload-results
```

```
<truncated>
2024-12-17T08:21:19.8104498Z Running ORT Scanner... 
2024-12-17T08:21:19.8190160Z Agent pid 2687
2024-12-17T08:21:19.9852173Z unknown shorthand flag: 'X' in -Xmx5120m
2024-12-17T08:21:19.9853018Z See 'docker run --help'.
2024-12-17T08:21:19.9879571Z ##[error]Process completed with exit code 125.
<truncated>
```

Analyzer runs fine, though.

[0_foss-scan _ ort.txt](https://github.com/user-attachments/files/18163592/0_foss-scan._.ort.txt)

--- a/action.yml
+++ b/action.yml
@@ -429,7 +429,7 @@ runs:
       if: contains(inputs.run, 'scanner')
       run: |
         echo -e "\e[1;33m Running ORT Scanner... "
-        eval "$(ssh-agent -s)" \
+        eval "$(ssh-agent -s)"
         docker run \
         --mount type=bind,source=$HOME,target=/home/ort \
         --mount type=tmpfs,target=/tmp \

